pdfarranger/core.py

```diff
@@ -292,7 +292,7 @@ def __init__(self, filename, basename, stat, tmp_dir, parent):
                         imgio.seek(0)
                         f.write(img2pdf.convert(imgio))
```

Comment:
This shouldn't need the `rotation` argument right?

Reply:
No, it's not needed here. If a fix is needed it will be a few line above at `bg.paste(img, mask=img.split()[-1])`. If we have no way to test it I suggest to ignore this case for now.